### PR TITLE
[14.0] [mig] pos_invoicing

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

File: pos_invoicing/__manifest__.py
Comment (ivantodorovich) [inline pos_invoicing/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: pos_invoicing/models/account_move.py
Comment (ivantodorovich) [inline pos_invoicing/models/account_move.py]: This method doesn't exist in upstream anymore, AFAICS

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Thanks! left some comments :)
Also please check pre-commit

**Comment by fshah-initos** [comment]
Hi @ivantodorovich
Can you please review the checks that are passed.

File: pos_invoicing/models/pos_session.py
Comment (ivantodorovich) [inline pos_invoicing/models/pos_session.py]: You can remove `_logger` from this file, as it isn't used

File: pos_invoicing/models/pos_session.py
Comment (ivantodorovich) [inline pos_invoicing/models/pos_session.py]:
```suggestion
        self.order_ids.account_move.pos_pending_payment = False
```

This is the same and more readable probably

File: pos_invoicing/models/account_move.py
Comment (ivantodorovich) [inline pos_invoicing/models/account_move.py]:
```suggestion
```

File: pos_invoicing/models/account_move.py
Comment (ivantodorovich) [inline pos_invoicing/models/account_move.py]:
```suggestion
        invoices = self.filtered("pos_pending_payment")
```
same & more readable than lambda

File: pos_invoicing/models/account_move.py
Comment (ivantodorovich) [inline pos_invoicing/models/account_move.py:22]: Can you add a test case for this?
Please use `assertRaisesRegex` to check that the right exception is raised

File: pos_invoicing/models/account_payment.py
Comment (ivantodorovich) [inline pos_invoicing/models/account_payment.py:17]: Can you add a test case for this?

File: pos_invoicing/models/account_payment.py
Comment (ivantodorovich) [inline pos_invoicing/models/account_payment.py]: Same, could you add a test case for this too?

File: pos_invoicing/tests/test_module.py
Comment (ivantodorovich) [inline pos_invoicing/tests/test_module.py]: Please do this in `setUpClass` instead.

File: pos_invoicing/tests/test_module.py
Comment (ivantodorovich) [inline pos_invoicing/tests/test_module.py]:
```suggestion
```
duh!

File: pos_invoicing/tests/test_module.py
Comment (ivantodorovich) [inline pos_invoicing/tests/test_module.py:62]: Move this helper up, right after `setUpClass`

**Review by ivantodorovich — COMMENTED** [review]
Thanks @fshah-initos !

Code coverage is still pretty low, do you think you could add some more test cases?
I'm not blocking on this, but it'd be really good if you do :)

Thanks!

File: pos_invoicing/models/pos_session.py
Comment (ikapasi-initos) [reply pos_invoicing/models/pos_session.py]: Already removed _logger in this file.

File: pos_invoicing/models/pos_session.py
Comment (ikapasi-initos) [reply pos_invoicing/models/pos_session.py]: This change is already implemented can you check it?

**Comment by ikapasi-initos** [comment]
@ivantodorovich Can you please review again this pull?

**Comment by dsolanki-initos** [comment]
Maintainers team, Could you please reopen this PR?

**Review by ivantodorovich — APPROVED** [review]

**Pull request closed without merging**
